main.go

```diff
@@ -28,6 +28,9 @@ func main() {
 	}
 	rootCmd.PersistentFlags().StringVarP(&logLevel, "log-level", "l", "info", "configures the logging level")
 	rootCmd.AddCommand(cmd.NewExecCmd())
+	rootCmd.AddCommand(cmd.NewExecAllCmd())
```

Comment:
Just noting that in theory exec cold have an argument `--all` and same with clean... but I'm not sure it matters, but it would reduce the number of top level commands.

Reply:
I agree. I decided to adhere to Terragrunt's model, which has entirely new/separate top level commands when executing for multiple levels within a hierarchy. I did the same since tfgen is meant to be a sort of alternative. However, I think we could easily move it to a `--all` flag if we decide it makes more sense or if tfgen starts getting a bit noisy/busy.